### PR TITLE
Content-Updates-1

### PR DESCRIPTION
**Pull request by anmol5936**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [ ] Make sure you are making a pull request against the **default branch** (left side). Also you should start _your branch_ off _default branch_.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.
- [ ] I have added necessary documentation (if appropriate)

### Description

Please describe your pull request.

❤️Thank you!

### Post merge checklist

- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by Shurtu-gal** [comment]
TODO:- 
- [ ] Sponsors
- [ ] About US
- [x] FAQ
- [x] EventPage ( Photos )
- [x] Footer
- [ ] Payment ( Contact and terms and conditions )

@starkjay21 @anmol5936

File: config/content/FAQ.js [hunk 1]
```diff
@@ -2,41 +2,52 @@ const FAQContent = {
...
       id: 7,
-      question: 'What is INNOVISION?',
-      answer: 'INNOVISION is the largest techno-management fest of Eastern India',
+      question: 'Cost of registration? ',
+      answer:
+        'The cost of registration, including accommodation is Rs. 950,' +
+        ' and without accommodation, it is Rs. 500.',
     },
     {
       id: 8,
```
Comment (jayeshxn) [inline config/content/FAQ.js]: Remove last object i.e. one with id: 8, it is repeated

File: config/content/FAQ.js [hunk 2]
```diff
@@ -2,41 +2,52 @@ const FAQContent = {
...
+        ' with participation from all over India ',
     },
     {
       id: 2,
-      question: 'What is INNOVISION?',
-      answer: 'INNOVISION is the largest techno-management fest of Eastern India',
+      question: 'Who are we?',
+      answer:
+        'Organised by students of NIT Rourkela, INNOVISION is the annual techno management.' +
+        'With our unrivalled ability to host such technical fests' +
```
Comment (jayeshxn) [inline config/content/FAQ.js]: add a space at the end of string in line 16

File: config/content/FAQ.js [hunk 3]
```diff
@@ -2,41 +2,52 @@ const FAQContent = {
...
+        "we were recognised as  Odisha's top student chapter",
     },
     {
       id: 3,
-      question: 'What is INNOVISION?',
-      answer: 'INNOVISION is the largest techno-management fest of Eastern India',
+      question: 'Who can participate?',
+      answer:
+        'All science and technology enthusiasts currently' +
```
Comment (jayeshxn) [inline config/content/FAQ.js]: add a space after end of string in line 23

**Review by jayeshxn — DISMISSED** [review]
follow the commit message convention

**Review by Shurtu-gal — APPROVED** [review]
LGTM 🤖

**Pull request merged**